backend/data/raw/lexicon-thucydideum/style.css

```diff
@@ -0,0 +1,15 @@
+head {
```

Comment:
This follows the pattern from #143 where we use https://github.com/TEIC/CETEIcean and a custom CSS stylesheet to display entries.

It was superseded by 68f76d085d3968cf4b6780adf769821cf69d1dcc because I wanted to make the `bibl` refs clickable.

Longer-term, I would like to revisit this and move away from "inline" styles provided by the XSLT.